sdk/servicebus/Azure.Messaging.ServiceBus/tests/Processor/SessionProcessorLiveTests.cs

```diff
@@ -890,6 +890,7 @@ async Task ProcessMessage(ProcessSessionMessageEventArgs args)
                     await Task.Delay(lockDuration.Add(lockDuration));
                     var lockedUntil = args.SessionLockedUntil;
                     Assert.AreEqual(lockedUntil, args.SessionLockedUntil);
+                    Assert.IsTrue(args.SessionLockCancellationToken.IsCancellationRequested);
```

Comment:
I can do that but it is going to take a few days until I have proper time. Otherwise feel free to push it here in the meantime. You should have editor rights

Reply:
Sounds good - I will add it.